README.md

```diff
@@ -73,6 +73,11 @@ see [Migrate from the SignalFx Java Agent](https://quickdraw.splunk.com/redirect
 
 <!-- Comments, spacing, empty and new lines in the section below are intentional, please do not modify them! -->
 <!--DEV_DOCS_WARNING-->
+<!--DEV_DOCS_WARNING_START-->
+The following documentation refers to the in-development version of `splunk-otel-java`. Docs for the latest version ([v2.0.0-alpha](https://github.com/signalfx/splunk-otel-java/releases/latest)) can be found [here](https://github.com/signalfx/splunk-otel-java/blob/v2.0.0-alpha/README.md).
```

Comment:
```suggestion
The following documentation refers to the in-development version of `splunk-otel-java`. Docs for the latest version ([v1.30.1](https://github.com/signalfx/splunk-otel-java/releases/latest)) can be found [here](https://github.com/signalfx/splunk-otel-java/blob/v1.30.1/README.md).
```

Reply:
I'd leave this as it is, the 1.x branch will point to 1.x docs. Though if you think this is needed then we can do it this way too.

Reply:
Nah I think you're right. Let main be main and 2.0 even tho deemed alpha. Thanks for talking me down.